templates.js

```diff
@@ -439,6 +449,13 @@ fa.sendEvent(formAnalytics.event.{{eventName}});
             'Example: `{"name": "First name", "surname": "Last name"}`.',
           default: '{}'
         },
+        {
+          id: 'iframeTracking',
+          type: 'boolean',
+          displayName: 'Send messages from iframes',
+          description: 'If checked, you won’t send the _paq.push but instead you will send a message to the parent window',
```

Comment:
```suggestion
          description: 'If checked, you will pass messages to the parent window instead of _paq.push',
```

Reply:
Sounds good, but I copied description from other templates so I would have to change them too.
@kbigos What do you think? Should we stick with previous description or should we change them to this one?

Reply:
Yeah I guess we can change them, it sounds better.